data/json/monstergroups/wilderness.json

```diff
@@ -2861,7 +2869,11 @@
         "starts": 360,
         "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
       },
-      { "monster": "mon_mutant_experimental", "freq": 2, "cost_multiplier": 6 }
+      { "monster": "mon_mutant_experimental", "freq": 2, "cost_multiplier": 6 },
+      {"monster": "mon_mantis_small", "freq": 2, "cost_multiplier": 8},
+      {"monster": "mon_aphid", "freq": 10, "cost_multiplier": 0},
```

Comment:
If we're going with aphids feeding on bushes/trees a swamp would be pretty bare for them.
```suggestion
```